SmalltalkSources/SBE-Streams.package/monticello.meta/version

```diff
@@ -0,0 +1 @@
+(name 'SBE-Streams-pre.3' message 'Adds a dot in one stream listing' id '96361132-952e-1c4e-bf56-a47679ff006c' date '3 December 2020' time '3:46:15.949768 pm' author 'pre' ancestors ((name 'SBE-Streams-pre.2' message 'Some code style improvements in SBE-Streams' id '58f39f8a-c1e3-814a-bd12-b54e3dea52d1' date '3 December 2020' time '1:58:35.193768 pm' author 'pre' ancestors ((name 'SBE-Streams-cypress.1' message 'fabricated from a Cypress format repository' id 'ba4e7e9e-99ff-4783-814c-da7997ae53b4' date '27 October 2020' time '5:55:06.974301 pm' author '' ancestors () stepChildren ())) stepChildren ())) stepChildren ())
```

Comment:
Did you change the type of the versioning system on purpose? I thought we're using Squot here.

Reply:
Hm... no. I am not used to the setup and used Filetree out of laziness. Does the changed version file break Squot? I took care not to delete the squot metadata.

Reply:
I don't know if FileTree is backward compatible with Squot, but at least it will become quite confusing as soon as someone commits via Squot and the `monticello.meta` files are out of sync ... But you could simply delete this folder again :-)